modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/skin/Utils.java

```diff
@@ -417,12 +417,14 @@ public static String computeClippedText(Font font, String text, double width,
     }
 
     public static String computeClippedWrappedText(Font font, String text, double width,
-                                            double height, OverrunStyle truncationStyle,
+                                            double height, double lineSpacing, OverrunStyle truncationStyle,
                                             String ellipsisString, TextBoundsType boundsType) {
         if (font == null) {
             throw new IllegalArgumentException("Must specify a font");
         }
 
+        height += lineSpacing;
```

Comment:
Modifying 'height' parameter seems incorrect as this parameter is used in other calculations below. If needed, use a separate local variable to use 'height+lineSpacing'

Reply:
The height used needs to be changed everywhere.  However, I can move this to the caller instead, with a comment explaining why:

> The computeClippedWrappedText call works with full lines (height of line + line spacing), including the last line.  However the wrap height does not include the line spacing for the last line.  In order to avoid wrapping the last line of text even though there is sufficient space the wrap height passed to computeClippedWrappedText is increased with the line spacing so it computes the correct text.

Reply:
I see your point. I am OK with the changes that you have done.